api_tests/swap.schema.json

```diff
@@ -1,375 +0,0 @@
-{
```

Comment:
We might want to keep the swap.schema tests?

Reply:
Done here: https://github.com/comit-network/comit-rs/pull/2855